app/assets/javascripts/pageflow/editor/views/theme_view.js

```diff
@@ -0,0 +1,67 @@
+/** @api private */
+pageflow.ThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'pageflow/editor/templates/theme',
+  tagName: 'li',
+
+  ui: {
+    name: '.theme_name',
+    selectButton: '.select_theme',
+    selectedRegion: '.theme_selected'
+  },
+
+  events: {
+    'click .select_theme': function(event) {
+      this.entry.configuration.set('theme_name', this.model.get('name'));
+      pageflow.updateStylesheet('theme');
+    },
+    'mouseenter': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == that.model.get('name');
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName());
+    },
+    'mouseleave': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == pageflow.entry.configuration.attributes.theme_name;
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName({
+        name: pageflow.entry.configuration.attributes.theme_name
+      }));
+    }
```

Comment:
Depending on DOM-elements of other views is an anti-pattern. This logic already exists in `ChangeThemeDialogView` and is duplicated here. Instead, `ChangeThemeDialogView` and `ThemeView` could share a `selection` Backbone.Model which is updated by the `ThemeView` and used by the dialog view to decide which preview to display. See [the `FilesExplorerView`](https://github.com/codevise/pageflow/blob/master/app/assets/javascripts/pageflow/editor/views/files_explorer_view.js#L24) for an example.